newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -71,8 +64,11 @@ func resourceNewRelicNrqlAlertCondition() *schema.Resource {
 							Required: true,
 						},
 						"since_value": {
-							Type:     schema.TypeString,
-							Required: true,
+							Deprecated:    "use `evaluation_offset` attribute instead",
```

Comment:
Just to close the loop here in case we reference this in the future. The `since_value` attribute name ended up being a poor choice for an attribute name because it's technically not a `SINCE` value in the NRQL query itself because the `SINCE` keyword is not supported for NRQL alert queries. So the new attribute is now properly named as `evaluation_offset`.  

**TODO:**  ensure our TF docs are clear about this